src/Organic/Organic.php

```diff
@@ -277,6 +284,31 @@ public function init( $apiUrl = null, $cdnUrl = null ) {
         add_action( 'save_post', [ $this, 'handleSavePostHook' ], 10, 3 );
     }
 
+    /**
+     * @param OrganicSdk $sdk
+     */
+    public function initSentry( \Organic\SDK\OrganicSdk $sdk ) {
+        if ( $this->environment != 'PRODUCTION' ) {
+            return;
+        }
+        try {
+            $config = $sdk->queryWordPressConfig();
+        } catch ( \Exception $e ) {
+            self::captureException( $e );
```

Comment:
`self` references the class and `this` references the "current" instance of the class; in this case, since `captureException` is a static method (a method on the class itself), I believe we cannot use `this`. (It isn't even an option in my IDE, for instance.) I could use `static` instead of `self`.

Reply:
I see. It looks like I can also use `$this::captureException`. From what I can tell, though, this is a nonstandard usage of $this.

Reply:
Ohh, I see... Yeah, makes sense!